packages/polymath-js/src/contracts/STO.js

```diff
@@ -48,13 +49,33 @@ export default class STO extends Contract {
       startTime,
       endTime,
       cap,
-      rate,
+      rawRate,
       fundsRaised,
       investorCount,
       tokensSold,
       isPolyFundraise,
     ] = this._toArray(await this._methods.getSTODetails().call());
 
+    const factoryAddress = await this._methods.factory().call();
+
+    const GenericModuleFactory = new Contract._params.web3.eth.Contract(
+      IModuleFactoryArtifacts.abi,
+      factoryAddress
+    );
+
+    // NOTE @RafaelVidaurre: This is a hacky implementation to be backwards-compatible
+    // with 0.0.0 version of the Capped STO. This will be tackled properly by
+    // module version supporton the SDK
+    const version = (await GenericModuleFactory.methods
+      .getUpperSTVersionBounds()
```

Comment:
This should be a call to `version`

Reply:
@monitz87 `version` doesn't exist in `0.0.0`. All I couldn found was `getUpperSTVersionBounds`

Reply:
`getUpperSTVersionBounds` gets the highest security token version that supports that module, it doesn't get the module version. Also, I'm looking at 2.0 `ModuleFactory.sol` and I see `string public version` defined

Reply:
@monitz87 will give it another look, I looked for the same thing since I share your concerns with `getUpperSTVersionBounds`